P/Pangomm/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pangomm"
+version = v"2.49.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/pangomm/$(version.major).$(version.minor)/pangomm-$(version).tar.xz",
+                  "a2272883152618fddea016a62f50eb23b9b056ab3c08f3b64422591e6a507bd5")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/pangomm*/
+if [[ "${target}" == "${MACHTYPE}" ]]; then
+    # Delete host libexpat.so to avoid confusion 
+    rm /usr/lib/libexpat*
+fi
+mkdir output && cd output
+meson --cross-file=${MESON_TARGET_TOOLCHAIN} ..
+ninja -j${nproc}
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct(["libpangomm-$(version.major)", "libpangomm-$(version.major).$(version.minor)"], :libpangomm)
```

Comment:
The version of the package is 2.49 but the library has name 2.48.  Sigh
```suggestion
    LibraryProduct(["libpangomm-$(version.major)", "libpangomm-$(version.major).48"], :libpangomm)
```